### ISSUE TITLE
About forward_cam

### ISSUE THREAD

**Issue by meiluzhu** [allenwu97/EDAM#6]
def forward_cam(self, x):
        x = super().forward(x)
        mask = self.mask_layer(x)
        mask = F.normalize(mask)
        mask = torch.abs(mask)
        return mask
=====================
Dear authors, I consider it might be unfair to use 'torch.abs(mask)'. Because the negative values corresponding to under-activated regions would be directly transformed into postive values. In fact,  'torch.relu(mask)' was used in previous works.

**Comment by allenwu97** [comment]
For masks which are generated by DA Layer, negative values are not corresponding to under-activated regions, but small values are. For each categories, we are not sure whether the activation regions should be positive values or negative values, because there still has a Classify Layer behind. In practice, approximately half of the categories have active regions corresponding to positive values, and the other half have negative values. If you use 'torch.relu(mask)', half of the categories‘ mIoU will be close to 0.

**Comment by meiluzhu** [comment]
Got it! Thank you very much!
